server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -111,7 +115,9 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
             }
 
             videos = likedVideoRes.getAsJsonArray("items");
-            genreAnalysis.calculateMusicCount(videos);
+            // videosRetrieved keeps track of music video order in genreAnalysis
```

Comment:
You don't need this comment since you already have a comment defining what this variable is when you initiated it